### PR TITLE
incorporate CSS styles from latest Pandoc's tempalte (ioslides)

### PR DESCRIPTION
**Pull request by atusy**
reflecting https://github.com/rstudio/rmarkdown/pull/1878#pullrequestreview-498085930

### CODE REVIEW

File: inst/rmd/ioslides/default.html [hunk 1]
```diff
@@ -92,6 +92,22 @@
...
+  ul.task-list{list-style: none;}
+  $if(quotes)$
+  q { quotes: "“" "”" "‘" "’"; }
+  $endif$
+  $if(highlighting-css)$
+  $highlighting-css$
+  $endif$
+  $if(displaymath-css)$
+  .display.math{display: block; text-align: center; margin: 0.5rem auto;}
+  $endif$
```
Comment (cderv) [inline inst/rmd/ioslides/default.html:110]: Are you sure this has to be added inside this selector ? 
I believe those styles is for the whole document instead, right ? 

Just a bracket issue I think

File: inst/rmd/ioslides/default.html [hunk 2]
```diff
@@ -92,6 +92,22 @@
...
 }
```
Comment (cderv) [inline inst/rmd/ioslides/default.html:111]: Are you sure this has to be added inside this selector ? 
I believe those styles is for the whole document instead, right ? 

Just a bracket issue I think

Comment (atusy) [reply inst/rmd/ioslides/default.html]: Oops!! Thanks for the review. Should be fixed now via [`438579c` (#1908)](https://github.com/rstudio/rmarkdown/pull/1908/commits/438579c9a84eb3b387cd38a70552d2d06f263bd3)

**Review by yihui — APPROVED** [review]
Thanks!

**Pull request merged**